backend/courses/views.py

```diff
@@ -376,3 +385,130 @@ def get_queryset(self):
             section__course__semester=get_current_semester(),
             in_add_drop_period=True,
         ).order_by("created_at")
+
+def get_accepted_friends(user):
+    """ Return user's accepted friends"""
+    Friendship.objects.filter(
+        Q(sender=user) | Q(recipient=user), 
+        status=Friendship.Status.ACCEPTED
+    ).values()
+
+class FriendshipViewSet(viewsets.ViewSet):
```

Comment:
Let's add a docstring to this class with some API documentation for each of the methods (GET, POST, DELETE) - I think this is necessary in this case because the flow is a bit unintuitive. This documentation will show up in our auto-docs under each of the new routes.

```python
"""
list: ...

create: ...

delete: ...
"""
```